.eslintrc.cjs

```diff
@@ -60,6 +60,14 @@ module.exports = {
     "@typescript-eslint/no-empty-function": "off",
     "@typescript-eslint/no-extra-semi": "warn",
   },
+  overrides: [
+    {
+      files: ["apps/portals/e2e/*.ts"],
+      rules: {
+        "@typescript-eslint/no-floating-promises": "error",
+      },
+    },
+  ],
```

Comment:
Lint e2e tests to find floating promises, as recommended in [Playwright best practices](https://playwright.dev/docs/best-practices#lint-your-tests)